roles/confluent.ksql/tasks/rbac.yml

```diff
@@ -85,6 +85,11 @@
             "resourceType":"Group",
             "name":"_confluent-ksql-{{ksql_final_properties['ksql.service.id']}}",
             "patternType":"PREFIXED"
+          },
+          {
+            "resourceType":"Topic",
+            "name": "_confluent-ksql-{{ksql_final_properties['ksql.service.id']}}",
```

Comment:
With this in place, there's no need to grant permissions to the `_confluent-ksql-{{ksql_final_properties['ksql.service.id']}}_command_topic` topic above (line 76) - but doesn't have to be done in this PR

Reply:
Removed.